moped-editor/src/views/staff/StaffForm.js

```diff
@@ -212,6 +225,101 @@ const StaffForm = ({ editFormData = null, userCognitoId }) => {
     });
   };
 
+  /**
+   * Makes sure the password looks ok
+   * @returns {boolean}
+   */
+  const passwordLooksGood = () =>
+    new RegExp(
+      "^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9])(?=.*?[#?!@$%^&*-]).{8,}$"
```

Comment:
Sweet regex! 

Maybe something like this npm module [Password Validator](https://www.npmjs.com/package/password-validator) might be helpful for things like this?